templates/faq.html

```diff
@@ -0,0 +1,79 @@
+{% extends "default.html" %}
+
+{%- block head -%}
+{%- endblock head -%}
+
+{% block content %}
+<div class="styled-title">
+    <h1 class="faq-title">Frequently Asked Questions</h1>
+</div>
+
+<div class="left-aside-bg">   
+    <div class="left-aside faq-aside">    
+        <aside>
+            <h2>CATEGORIES</h2>
+            <ul class="category-nav">
+                <li><a href="#general" class="category-link">General</a></li>
+                <li><a href="#memory-performance" class="category-link">Memory & Performance</a></li>
+                <li><a href="#technical" class="category-link">Technical</a></li>
+                <li><a href="#limitations" class="category-link">Limitations</a></li>
+            </ul>
+        
+            <div class="edit-box">
+                <span>Found an Issue?</span>
+                <a href="https://github.com/valkey-io/valkey-io.github.io/edit/main/content/topics/faq.md" class="edit-link" target="_blank">
+                    Edit this page on GitHub
+                </a>
+            </div>
+        </aside>
+
+        <main class="container">
+            {% if config.extra.review_list is containing(page.path) %}
```

Comment:
I don't think we need the review list bits. This appears to be duplicative from the `docs-page.html` file which only exists as a function of importing a bunch of dubious content. This isn't imported so I think we can drop all that.